docs/configuration.md

```diff
@@ -46,6 +46,11 @@ project_name: "my-repo"
 # that might be used by the consumer of the Salus report.
 custom_info: "PR-123"
 
+# Numeric (Integer or Float)
+# Defines how many seconds Salus should run its scans before timing
+# out (defaults to indefinite amount of time if not specified)
+salus_timeout_s: 120
```

Comment:
My guess is that this `salus_timeout_s` will override `scanner_timeout_s`?
If so, could you clarify this in the doc?

Please test out using `scanner_timeout_s` and `salus_timeout_s` in the same salus.config to ensure there are no strange issues.
Please use `docker build ...` then `docker run ...` to test this, and test different combinations of `salus_timeout_s` and `scanner_timeout_s`, something like `salus_timeout_s` < `scanner_timeout_s` and  `salus_timeout_s` > `scanner_timeout_s`